charts/pulsar/templates/broker-statefulset.yaml

```diff
@@ -169,6 +169,7 @@ spec:
           echo "OK" > status;
           {{- include "pulsar.broker.zookeeper.tls.settings" . | nindent 10 }}
           bin/pulsar zookeeper-shell -server {{ template "pulsar.zookeeper.connect" . }} get {{ template "pulsar.broker.znode" . }};
+          cp /logconf/log4j2.yaml /sn-platform/conf;
```

Comment:
why not just mount the log4j2.yaml directly to the target log4j2 configuration file without copying the file?

Reply:
If I directly mount the `log4j2.yaml` file under `/sn-platform/conf`, the other files in this folder will be overwritten.